uncrustify.cfg

```diff
@@ -1981,7 +1981,7 @@ nl_split_while_one_liner        = false    # true/false
 #
 
 # The maximum number of consecutive newlines (3 = 2 blank lines).
-nl_max                          = 0        # unsigned number
+nl_max                          = 2        # unsigned number
 
 # The maximum number of consecutive newlines in a function.
 nl_max_blank_in_func            = 0        # unsigned number
```

Comment:
```suggestion
nl_max                          = 3        # unsigned number

# The maximum number of consecutive newlines in a function.
nl_max_blank_in_func            = 2        # unsigned number
```

Reply:
This would leave room to have either 1 or 2 blanks before things like function or class definitions, which IMO should be fixed to either 1 (I just used 1 because our code base does more or less that) or 2 (as for instance pep8 enforces for python). What I would not do is to leave it up to the contributor, I'd rather have some uniformity

Reply:
Yes, definitely! That's why I'd like as many opinions as possible here